config/application.yml.example

```diff
@@ -35,7 +35,12 @@ on_application_denial:
   # For instance, you may wish to provide feedback on steps applicants
   # could take in order to become eligible for the position in the future.
   notify_of_reason: true
-
+  # Whether or not an applicant will be allowed to resubmit an application
+  # to a declined application
```

Comment:
I'd go with "whether or not an applicant will be allowed to resubmit an application for a particular position when they have previously had an application for that position denied"